app/server.R

```diff
@@ -932,7 +946,7 @@ server <- function(input, output, session) {
     }
     asOfChoices <- c(asOfChoices, CURRENT_WEEK_END_DATE())
     # Make sure we have a valid as of selection
-    nonValidAsOf <- selectedAsOf == "" || !(as.Date(selectedAsOf) %in% asOfChoices)
+    nonValidAsOf <- selectedAsOf == "" || !(selectedAsOf %in% asOfChoices)
```

Comment:
confirming: we don't need to cast `selectedAsOf` to Date anymore?

Reply:
This was actually causing a problem when trying to view non-current as-of dates, so I'm reverting. Thanks for questioning this change!

In R, dates can be compared to strings for equality (`==`, `>`, etc), which can save time by avoiding having to convert from string to date, but apparently the same isn't true of the `%in%` operator.